src/main/webapp/i18n/de/plagiarism.json

```diff
@@ -37,8 +37,9 @@
             "minimumSizeTooltipModeling": "Berücksichtige nur Einreichungen, die mindestens so viele Modeling-Elemente wie der angegebene Wert haben",
             "minimumSizeTooltipText": "Berücksichtige nur Einreichungen, die mindestens so viele Zeichen wie der angegebene Wert haben",
             "noInfoAvailable": "Keine Information verfügbar",
-            "caution": "Die Plagiatskontrolle kann für große Kurse sehr lange dauern.",
-            "detect": "Plagiate finden",
+            "caution": "Die Plagiatskontrolle kann sehr lange dauern.",
+            "configurationMoved": "Die Konfiguration wurde auf die Seite der Übungsaktualisierung verschoben.",
```

Comment:
That is really tough to understand.

Probably be easier just to give a link. to follow.

Reply:
This was agreed with @MarkusPaulsen to be like that. After this feature matures we will anyway remove that information

Reply:
Then still make it clearer, please. I would have no idea what "Übungsaktualisierung" means.

Reply:
Then I need some guidance for better translation of _Configuration was moved to the exercise update page._. Do you have any ideas?

Reply:
"Bearbeitungsseite der Übung/Aufgabe" maybe?

Reply:
Done